src/crs.jl

```diff
@@ -156,3 +156,4 @@ end
 include("crs/basic.jl")
 include("crs/geographic.jl")
 include("crs/projected.jl")
+include("crs/rand.jl")
```

Comment:
In CRS.jl we try to keep all functions near the CRS types so that contributors can find which methods are missing.

In this case we just need to add two `rand` methods below each CRS definition:

```julia
rand([rng], CRS{Datum}, [n]) = # creates concrete instance(s)
rand([rng], CRS, [n]) = # creates concrete instance(s) with default datum of CRS
```